### PR TITLE
Update time and added streaming check capabilities

### PR DESCRIPTION
**Pull request by scottdj92**
canistreamit package provides a check to see whether or not a movie is
available to stream on Netflix, Amazon, Hulu, etc…

### CODE REVIEW

**Comment by scottdj92** [comment]
Currently the canistreamit package only logs out the variable for testing. It is not included in the bot anywhere until we can figure out what is the best way to implement this behavior.

**Comment by scottdj92** [comment]
**OPTION 1**
Fire the event when the winning movie is selected.

**OPTION 2**
Fire the event whenever someone submits a vote. This may cause performance issues and a cluttering of the channel.

**Comment by dxn7335** [comment]
Might be good to only fire event when the winning movie is selected and include it in a winning movie message or when someone runs a command "canstream {movie}" if they are curious

File: lib/usherbot.js [hunk 1]
```diff
@@ -108,6 +109,18 @@ u.handleVote = function( movie ){
 	return this.messages.success.vote.replace( "{movie}", movie );
 }
 
+// CHECK TO SEE IF MOVIE IS AVAILABLE TO STREAM
+u.searchAvailableStream = function( movie ) {
+    StreamIt.search(movie)
+    .then(function(results) {
+        //return StreamIt.streaming(results[0]);
+        console.log(results);
```
Comment (dxn7335) [inline lib/usherbot.js:117]: What does this return? (string, array, etc)
We can probably just add a string into our message content file for stream movies like "This movie can be streamed on " and append whatever results onto it.

Comment (scottdj92) [inline lib/usherbot.js:117]: Yeah, I haven't had the time to test it out. Judging from the docs, it looks like it returns an object.

[package docs](https://www.npmjs.com/package/canistreamit)

**Comment by scottdj92** [comment]
I think adding a canstream command is better so we have something that we can fire "on-demand" and also when the winning movie is decided. But should we reject movies based on their streaming capabilities?

**Pull request closed without merging**